From bd4191b2b06e1558b87a3ab5b0e17572f7633edb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <sebastian@centricular.com>
Date: Mon, 3 Jul 2017 09:09:25 +0300
Subject: [PATCH] registrybinary: Update magic version string

This forces a regeneration of the registry and unblacklists anything
that was blacklisted before if possible.

See

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=866339
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=866103
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=819594#26
---
 gst/gstregistrybinary.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/gstregistrybinary.h b/gst/gstregistrybinary.h
index 22e56d19..f98e3441 100644
--- a/gst/gstregistrybinary.h
+++ b/gst/gstregistrybinary.h
@@ -55,7 +55,7 @@ G_BEGIN_DECLS
  * This _must_ be updated whenever the registry format changes,
  * we currently use the core version where this change happened.
  */
-#define GST_MAGIC_BINARY_VERSION_STR "1.3.0"
+#define GST_MAGIC_BINARY_VERSION_STR "1.12.0"
 
 /*
  * GST_MAGIC_BINARY_VERSION_LEN:
-- 
2.13.2

